docs-user/guide-remote-profiling.md

```diff
@@ -1,28 +1,125 @@
+ - Fenix is supposed to become a GeckoView-based replacement for Fennec in the future, but at this point, it is in a very early stage and there might not even be downloadable builds for it.
+ - Lastly, there's the GeckoView-example app: This app is an extremely simple GeckoView-based app, which is generated along with every GeckoView build on the mozilla-central trees. That means you can get GeckoView-example apks for any GeckoView build you want. GeckoView-example is also the easiest GeckoView-based app to compile locally. It's not really usable as your daily browser, and it's not on the Play Store so it doesn't auto-update, but it's usually enough for performance testing and profiling work.
+
+So there are really only two good options: GeckoView-example and the reference browser. Both have to be installed manually by downloading an apk file to your phone; GeckoView-example has a slightly more up-to-date version of Gecko.
+
+### Download GeckoView example
+
+The latest GeckoView example Nightly always lives at this URL: [https://index.taskcluster.net/v1/task/gecko.v2.mozilla-central.nightly.latest.mobile.android-api-16-opt/artifacts/public/build/geckoview_example.apk](https://index.taskcluster.net/v1/task/gecko.v2.mozilla-central.nightly.latest.mobile.android-api-16-opt/artifacts/public/build/geckoview_example.apk)
+
+Here's a QR code for that URL:
+
+![https://index.taskcluster.net/v1/task/gecko.v2.mozilla-central.nightly.latest.mobile.android-api-16-opt/artifacts/public/build/geckoview_example.apk](./images/qr-gve-nightly.gif)
+
+### Alternative: Download the reference browser
+
+If you'd prefer to profile the reference browser instead, here's a QR code for the latest reference browser build, which always lives at [https://index.taskcluster.net/v1/task/project.mobile.reference-browser.nightly.latest/artifacts/public/reference-browser-arm.apk](https://index.taskcluster.net/v1/task/project.mobile.reference-browser.nightly.latest/artifacts/public/reference-browser-arm.apk):
+
+![https://index.taskcluster.net/v1/task/project.mobile.reference-browser.nightly.latest/artifacts/public/reference-browser-arm.apk](./images/qr-reference-browser-nightly.gif)
+
+### Installation
+
+In order to install an app from an apk file onto your Android phone, follow these steps:
+
+ 1. First, if you already have another build of this app installed, uninstall it. A new installation doesn't automatically replace an existing installation; the installation process will fail if an app with that name already exists. Uninstalling can be done by dragging the app's icon into a screen corner that displays a trash can.
```

Comment:
Note: that doesn't work like this on my Android, I need to go in the Settings or the Play Store to uninstall an app.